editor/components/document-outline/index.js

```diff
@@ -25,6 +25,10 @@ const incorrectLevelContent = [
 	<br key="incorrect-break" />,
 	<em key="incorrect-message">{ __( '(Incorrect heading level)' ) }</em>,
 ];
+const multipleH1Headings = [
+	<br key="incorrect-break-h1" />,
```

Comment:
We should fix these classes.

Reply:
Not sure I understand?

Reply:
oh, just that the namespace should be the component not an arbitrary class. This predates this PR, though.

Reply:
this is just the React "key" though not a classnamee :)

Reply:
Haha, welcome 2018.

Reply:
If we export `React.Fragment` from `element/index.js`, we could write this as:

```jsx
const multipleH1Headings = (
	<Fragment>
		<br />
		<em>{ __( '(Multiple H1 headings)' ) }</em>
	</Fragment>
);
```

(Not sure if others feel this is more or less readable)